src/geometry/manifold/algebra/left_invariant_derivation.lean

```diff
@@ -98,12 +98,10 @@ instance : has_add (left_invariant_derivation I G) :=
     left_invariant', pi.add_apply]⟩ }
 
 instance : has_neg (left_invariant_derivation I G) :=
-{ neg := λ X, ⟨-X, λ g, by simp only [linear_map.map_neg, derivation.coe_neg, left_invariant',
-    pi.neg_apply]⟩ }
+{ neg := λ X, ⟨-X, λ g, by simp [left_invariant']⟩ }
```

Comment:
For some reason I don't understand, `simp only [...]` timeouts here and in `has_sub`.